tt_eager/tt_dnn/op_library/composite/composite_ops.hpp

```diff
@@ -160,6 +160,11 @@ Tensor addcdiv(
     float value,
     const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
 
+Tensor binary_div(
```

Comment:
Is there a reason we are calling this op `binary_div`?
Our other eltwise binary ops (add, sub, mul) do not have `binary_` in the name?

Reply:
Since both the inputs are tensors , I've named it as binary_div, and moreover there is a seperate op `Tensor div_unary( const Tensor& input_tensor, float value, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);` . Hence the naming @tt-aho  .